code/modules/projectiles/guns/projectile/automatic/ak47.dm

```diff
@@ -1,8 +1,8 @@
 /obj/item/weapon/gun/projectile/automatic/ak47
 	name = "Excelsior 7.62x39 AKMS"
-	desc = "Weapon of oppressed, oppressors and just crazy terrorists.\
-		 If it doesn't work, you can always hit him with it! It is the really old designed assault rifle, chambered in 7,62x39.\
-		 It is known for it easy maintaining and low price. This gun is not used by military anymore, but it found a wide spread within criminals and insurgents."
+	desc = "Weapon of an oppressed, oppressors, and extremists of all flavours.\
+		 This is an ancient automatic rifle, chambered in 7.62x39. If it won't fire, percussive maintenance should get it working again.\
+		 It is known for its easy maintaining, and low price. This gun is not in active military service anymore, but has become ubiquitous among criminals and insurgents."
```

Comment:
this is still wrong, you only copied some of what i wrote before

should be "the oppressed"
"easy maintenance"